client/src/pages/games/GamesPage.tsx

```diff
@@ -4,37 +4,57 @@ import { Game } from "../../types/types";
 import GameItem from "./GameItem";
 import PageLayout from "../../components/common/PageLayout";
 import toast from "react-hot-toast";
+import PaginatedList from "../../components/common/PaginatedList";
+
+const PAGE_SIZE_LIMIT = 20;
```

Comment:
can make this dynamic but i dont see another use case right now other then games page